public/js/controllers/charts_controller.js

```diff
@@ -41,6 +42,10 @@ function isScaleDisabled (chart) {
   return lineScales.indexOf(chart) > -1
 }
 
+function isModeEnabled (chart) {
+  return modeScales.indexOf(chart) > -1
```

Comment:
I think `return modeScales includes(chart)` is a bit more expressive here.

Reply:
fixing it now

Reply:
this is done